launch/speech_recognition.launch

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<launch>
+
+    <!-- Get the machine file -->
+    <arg name="machine" default="localhost"/>
+    <include file="$(env ROBOT_BRINGUP_PATH)/machines/$(arg machine).machine" />
+
+    <!-- Always run a multicast server with a string topic answerer for amigo-hear -->
+    <node name="hmi" pkg="hmi" type="multi_client" machine="$(arg machine)" output="screen"/>
+
+    <!-- TODO Make path dynamic -->
```

Comment:
Please file issue for each `#TODO`

Reply:
I won't merge this branch without fixing this anyway. So it will be taken care of before the merge.